src/webextension/locales/en-US/list.ftl

```diff
@@ -52,11 +52,7 @@ all-items-empty =
 all-items-filtered = No results
 
 homepage-title =
-  { $count ->
-     [0]     Welcome to { product-title }
-     [1]     You have { $count } entry in your { product-title }
-    *[other] You have { $count } entries in your { product-title }
-  }
+  the simple way to store, retrieve, and manage website login info
```

Comment:
I think I commented on this on #362, but this doesn't appear to follow the Photon [spec for capitalization](https://design.firefox.com/photon/copy/capitalization.html). I looked around through the Firefox UI and I can't find any examples of all lower-case text like this. Pretty much everything is either title case or sentence case (with a smattering of all-caps stuff).